src/pages/[id].page.jsx

```diff
@@ -31,13 +31,15 @@ export default function Details({ details }) {
   // dynamic imports need to be asynchronous
   useEffect(() => {
     const updateCodeBlocks = async () => {
-      const html = await import(`../code-examples/${slug}/html.js`);
+      const html = await import(`../code-examples/${slug}/${slug}.html.js`);
       setHtmlCode(html.default);
 
-      const css = await import(`../code-examples/${slug}/css.js`);
+      const css = await import(`../code-examples/${slug}/${slug}.css.js`);
       setCssCode(css.default);
 
-      const javascript = await import(`../code-examples/${slug}/javascript.js`);
+      const javascript = await import(
+        `../code-examples/${slug}/${slug}.javascript.js`
```

Comment:
This is my fault for changing my mind and not updating the cards. We do want the code example files named the way they are in `main`. It's just slightly less redundant and makes it so we don't have to do this double interpolation with the `slug`.

Reply:
Alright, I can remove those. Sorry if I missed that conversation!
https://github.com/sparkbox/accessible-components/pull/77/commits/be3f5e376d755203b042b9ce39dcd6dd8f9a5aae